provider/v1alpha1/service.proto

```diff
@@ -59,6 +61,27 @@ message MountRequest {
 message MountResponse {
     repeated ObjectVersion object_version = 1;
     Error error = 2;
+    // files contains the entire mount volume filesystem.
+    //
+    // The total size of all files should not exceed 1MiB or syncing to
```

Comment:
Do we want to validate the file content size if this is a known limit?

Reply:
We could add it for the 1 MiB case but over 4MiB the RPC will fail. I've added a klog to `MountContent` for when the proto is > 1MiB.